src/main/resources/templates/browse.html

```diff
@@ -19,14 +19,15 @@
 
     <link href="https://cdn.syncfusion.com/ej2/ej2-layouts/styles/material.css" rel="stylesheet" type="text/css"/>
     <script src="https://cdn.syncfusion.com/ej2/ej2-layouts/dist/global/ej2-layouts.min.js" type="text/javascript"></script>
-
+    <!-- Local CSS and JS -->
     <link href="/css/main.css" rel="stylesheet" type="text/css"/>
     <script src="/js/browse.js" type="text/javascript"></script>
     <script src="/js/shared.js" type="text/javascript"></script>
 
-    <title>Home</title>
+    <title>TvTracker Browse</title>
 </head>
 <body onload="startUp()">
+<!-- Search bar for movies or series -->
```

Comment:
I wouldn't worry too much about commenting HTML.  This will be visible on the clients' browsers (via view source), after all.